.circleci/config.yml

```diff
@@ -190,7 +190,7 @@ jobs:
       - install-deps
       - go/mod-download
       - run:
-          command: make buildall
+          command: make build
```

Comment:
This should be buildall

Reply:
Is there any reason for it to be `builall`, we build `lotus` `lotus-storage-miner` and `lotus-seal-worker` which should check all possible problematic components on OSX.

It is still buildall on linux, I switched it to `build` because OSX is slow.